Header header
string[] arm_name
geometry_msgs/Point goal_position
geometry_msgs/Quaternion goal_orientation
string planner_name
int32 n_attempt
float32 c_time
int32 n_repeat
int32 n_maxstep
moveit_msgs/RobotState start_state
---
int64 w_flag
int64 feasibility
moveit_msgs/RobotTrajectory r_trj
